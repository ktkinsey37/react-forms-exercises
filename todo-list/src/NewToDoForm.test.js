import React from "react";
import { render, fireEvent } from "@testing-library/react";
import NewToDoForm from "./NewToDoForm";

it("renders without crashing", function() {
  render(<NewToDoForm />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<NewToDoForm />);
  expect(asFragment()).toMatchSnapshot();
});
