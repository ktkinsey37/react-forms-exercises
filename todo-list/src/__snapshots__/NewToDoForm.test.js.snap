// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form>
    <label
      for="todo"
    >
      ToDo:
    </label>
    <input
      id="todo"
      name="todo"
      placeholder="Add New Todo"
      value="New ToDo"
    />
    <button>
      Add a new ToDo!
    </button>
  </form>
</DocumentFragment>
`;
